### ISSUE TITLE
Notion app isn't open

### ISSUE THREAD

**Issue by farhannovan** [notion-enhancer/notion-enhancer#435]
FILL OUT THIS TEMPLATE, REMOVING/REPLACING ALL NON-BOLD TEXT WITH YOUR OWN INFORMATION.
BE AS CLEAR AND CONCISE AS POSSIBLE.

**describe the bug**
After disable the theme application cannot open, only the notion enhancer icon appears but there is no features 

**expected behaviour**
after I reinstalled it still couldn't be opened

**screenshots**
![Screenshot (31)](https://user-images.githubusercontent.com/74697034/109381490-45f2ae00-790d-11eb-8aef-4fb6a2c70363.png)

**platform**
windows 10

**Comment by cyboryan** [comment]
I also encountered the same issue after enabling a theme (it worked fine on minor tweaks without issues). On relaunch, the app suddenly closed and removed the app icon on start and taskbar, at least it didn't last long but when it reappeared, all tweaks are disabled and is back to what Notion looks like without an enhancer.

Reinstalling the enhancer will retain the issue as captured in the picture above by @farhannovan. Right-clicking the enhancer icon doesn't display the context menu. I also reinstalled Notion but it all remains the same.

Hopefully, this gets fixed. All the best!

**Comment by huriprasath** [comment]
I just had the same issue too. Everything was working fine until I disabled custom-inserts in the Enhancements window, and both Notion and Enhancements windows crashed. And when I tried to open Notion again, it launched without notion-enhancer.

So I tried reinstalling notion-enhancer, and now, Notion window simply refuses to launch with notion-enhancer enabled.
If notion-enhancer is applied, when I try to launch Notion, 4+1 background processes labelled "Notion" get added in the task manager, like in @farhannovan's screenshot. No window opens up, and but the Windows tray icon appears. But the tray icon neither opens Notion window on left click nor the context menu on right click.

Hope this gets fixed soon.
